csharp/ql/test/library-tests/cil/consistency/Handles.expected

```diff
@@ -3,12 +3,12 @@ tooManyMatchingHandles
 missingCil
 csharpLocationViolation
 matchingObjectMethods
-| Equals(object) | System.Boolean System.Object.Equals(System.Object) |
```

Comment:
Why is there a change in the printed signatures?

Reply:
The test was changed to only inspect DotNet callables. I will change the test to make a downcast instead. In any case, I haven't been that thorough as a follow up PR will delete all these.